### ISSUE TITLE
In VLC Media Player, when focus is on Open File... in Media submenu, and user presses right arrow and then left arrow NVDA behaves strangely

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4729]
**Reported by bhavyashah on 2014-12-21 04:39**
I will not be able to concisely explain this issue, only give a rough idea of its reproduction steps. I request the developers to investigate this further themselves after they are able to reproduce this.
Steps to reproduce
Launch VLC Media Player (latest stable version).
Press Alt to go to the menu bar, where your focus will be placed on the Media submenu and press down arrow once and you will land on Open File...
Now right arrow and listen carefully to what NVDA has to say. After right arrowing a couple of times, begin left arrowing and listen to NVDA carefully.
I cannot describe the actual result correctly, so developers may understand the actual result by reproducing this themselves.

**Comment by bhavyashah** [comment]
I can still reproduce this issue in VLC Media Player version 2.2.3 (although I took some time to comprehend what I meant by my rather vague bug report 🖌 ).
To elaborate somewhat, the problem is that when you are let's say on any item (provided it isn't a submenu in itself) of the Media submenu part of the menu bar of VLC, and you press the right or left arrow to move to the first item of the next or previous menu (Playback or Help respectively), the following happens:
Expected Result: Not sure. NVDA should ideally report the menu name followed by the menu item name, but only the latter would suffice as well.
Actual Result: NVDA starts to report the name of the menu but stutters midway because the speec is interrupted by 'menu'. In other words, when I press right arrow in the above example, NVDA says 'Playback' incompletely and then says 'unavailable' and then 'menu. NVDA speaks these individually with pauses akin to stuttering between them.
Additionally, this is not a VLC Media Player specific issue. I just replicated the same with Wordweb 8.0 in a similar manner, and I expect this to be the case with the menu bars of several other software.
This definitely is not the most earth-shattering or life-threatening of issues, but an annoyance I would at least request a quick investigation of, so as to understand the scope of this bug. @leonardder

**Comment by Adriani90** [comment]
@javidominguez, @bhavyashah can you still reproduce this issue in the newest VLC version?

**Comment by DataTriny** [comment]
Hello,
I can reproduce the mentionned behavior in VLC Media Player 3.0.4 on Windows 10 version 1809.

**Comment by bhavyashah** [comment]
I cannot reproduce this issue with NVDA 2023.2 and VLC media player version 3,0,18,0 or WordWeb Pro version 10.2.0.0. On arrowing left and right, the menu changes and NVDA speaks the menu item it moved to but not the menu itself. Maybe it'd be nice if NVDA reported the name of the menu it switched to, or maybe that is extraneous  given the user has other ways of knowing that? I don't have strong feelings on that either way and that is a somewhat distinct question from the app-specific bug reported here. Feel free to close this ticket.

**Comment by Adriani90** [comment]
> On arrowing left and right, the menu changes and NVDA speaks the menu item it moved to but not the menu itself.

This is exactly this issue, so it seems still reproducible. NvDA interupts speech when changing from a menu to another, so the reporting of the menu title is cut and the reporting of the focused menu item is prioritized.
Expected: NVDA should report the menu title when entering the menu with left or right arrow keys.

**Comment by Adriani90** [comment]
Closing as duplicate of #4278.
